service/qos/service_level_controller.hh

```diff
@@ -150,9 +150,10 @@ public:
      * Start legacy update loop if RAFT_SERVICE_LEVELS_CHANGE feature is not enable yet 
      * or the cluster is in recovery mode
      * or the cluster hasn't been migrated to raft topology.
+     * Otherwise, do initial update of service level configuration to fetch already existing data.
```

Comment:
Why is it necessary to do this? Won't we load the service levels configurations when we start group0?

Reply:
No, we don't. But I've moved this to topology_state_load()